docs/customizations/custom-server.md

```
@@ -0,0 +1,77 @@
+---
+aside: false
+outline: false
+---
+
+# Custom Server
+
+You can allow users to set a custom server URL in the Playground section.
+
+<ScopeConfigurationTabs>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: ScopeConfigurationTabs

(MD033, no-inline-html)